apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -217,14 +218,14 @@ func (a *Facade) UpdateApplicationsUnits(args params.UpdateApplicationUnitArgs)
 			continue
 		}
 
-		// always set to `false` to disable `jujuManagedUnits`
-		jujuManagedUnits := false
-		if jujuManagedUnits {
-			// TODO: remove here?
-			err = a.updateJujuManagedUnitsFromCloud(app, appUpdate.Units)
-		} else {
-			err = a.updateUnitsFromCloud(app, appUpdate.Units)
+		appConfig, err := app.ApplicationConfig()
+
+		// ingore `jujuManagedUnits`, log warning if it's still set to true
+		if appConfig.GetBool(caas.JujuManagedUnits, caas.JujuDefaultJujuManagedUnits) {
+			logger.Warningf("Config `%v` is DEPRECATED. Even current value is set to `true`, juju treats it as `false` now", caas.JujuManagedUnits)
```

Comment:
I think this would be a bit better as:
```
logger.Warningf("config value %q is deprecated - although it is set to true, juju will not manage kubernetes units", caas.JujuManagedUnits)
```

Reply:
yes, much better, updated